### DISCUSSION TITLE
Controller Config GUI don´t open

### DISCUSSION THREAD

**Discussion by iEnki** [SeriousOldMan/Simulator-Controller#6] [Q&A] [answered]
Hello There,

i dont know, if this is a Bug but i can´t re-open the Simulator Config.Exe after pressing the "Cancel" Button.
I was just looking to the Plugin Tab, so there was no Changes what i had to  save.

But now, the GUI dont wanna re-open again. I got the Black Symbol in may Task Tray in the right Botton from Windows, but no GUI to can made changes in the config.
Is there any Workaround for this? 
![sim](https://user-images.githubusercontent.com/12661967/114408971-3fad4e00-9baa-11eb-90b0-93786d6afe47.PNG)

Simulator Settings. exe etc. will start normaly. I also deleted your Tool , i think completely ans download again , but still the Same trouble :(

Thanks for Help.
Best regards

**Comment by SeriousOldMan** [comment]
Hi,

sounds strange. If you haven’t changed the configuration already, you can try to delete the file “Simulator Configuration.ini” in the “Simulator Controller\Config” folder which is located in your standard Windows “Documents” folder.

If this doesn’t help you can delete the “Simulator Controller” in your “Documents” folder completely, which is like starting a full fresh installation. Not a god solution, when already heavily configured, but in your case a good solution.

Best regards

Oliver

P.S. You can always close the Apps by using the right mouse menu on the Icons in the Task Tray.

**Comment by iEnki** [comment, marked as answer]
Hi Oliver,

Big Thanks for the fast response. Be Sure , i tryed this but still the same Problem . 
Iam confused because program workes fine last few days.  

Maybe the Error Log can help at this trouble ? Because i see, there a critical error that i can´t explain.

[Critical - 20210412170950]: ---------------------------------------------------------------
[Critical - 20210412170950]:       Running Simulator Configuration (2.8.0-release) [28432]
[Critical - 20210412170950]: ---------------------------------------------------------------
[Warn     - 20210412170950]: Installation path not set
[Warn     - 20210412170950]: AutoHotkey path not set
[Warn     - 20210412170950]: MSBuild path not set
[Critical - 20210412170952]: Unhandled exception encountered in C:\Users\enki8\Documents\Assetto Corsa Competizione\Simulator Controller\Binaries\Simulator Configuration.exe at line 5595: 0x8004503A - 
Source:		(null)
Description:	(null)
HelpFile:		(null)
HelpContext:	0

Best regards

Robert

**Reply by SeriousOldMan** [reply]
This is even more strange, since you are encountering a low level error in the Script Runtime of AutoHotkey. Is it possible, that an Antivirus tool has a problem with the AutoHotkey applications? Unfortunately, this is a common problem with AutoHotkey, as I have written in the README file.

You can try to install AutoHotkey on your system and run the "Simulator Configuration.ahk" from the "Sources\Tools" directory directly. Maybe we then get a better error message than in the compiled version.

Best regards

Oliver

**Reply by SeriousOldMan** [reply]
Another idea: Please ZIP the "Simulator Controller" in your "Documents" folder and send it to me, if you like. Maybe I can reproduce the error, if it has its root cause in a configuration issue.

**Reply by iEnki** [reply]
Hey Again, 

really thanks for your fast response, but i don´t wanna Waste your Time ! You need it for this Great Project ;)

I found the Problem ! It was my own stupidity.
Yesterday i tryed to add a better TTS Voice , because the tool don´t recognize not all my Voices from Windows. So i changed some regestry entry on my System to acivated it in your Tool. 
It looks like that mades the trouble, i made changes back and all be fine again. :)

I just was confused, because my changes has nothing to do with any file from your tool. But Ok, now i know 

Thanks for your Help !

Best regards

**Reply by SeriousOldMan** [reply]
Did you unblock all files in the Binaires folder (since you wrote, that you have trouble talking to Jona)?

**Reply by iEnki** [reply]
Yes i do ! 

I dont have Problems to talk with Jona. 
I wanted to change her TTS Voice....in Windows i got the Voice Stefan, but i can´t choose it in the Settings from the Simulator Config. 
So i tried this https://xcomputer.website/2018/08/11/schalte-alle-windows-10-tts-stimmen-system-weit-um-mehr-von-ihnen/  to get it Work with Jona . 
Iam a German User, so the Original Voices from WIndows (Hedda) is terrible, that is only my Trouble with Jona that i tryed to change.

**Reply by SeriousOldMan** [reply]
This is a problem, I also have (I am a German, too). If you install the English voices, they are all recognized, but for an unknown reason "Stefan" is different. Sorry, no solution for that from my side. :-)
